sources/packages/web/src/components/common/OfferingForm.vue

```diff
@@ -2,53 +2,105 @@
   <formio-container
     formName="educationProgramOffering"
     :formData="formData"
-    @submitted="saveOffering"
+    @loaded="$emit('loaded', $event)"
+    @changed="(form, event) => $emit('changed', form, event)"
+    @submitted="submitOffering"
   >
-    <template #actions="{ submit }" v-if="!readOnly">
+    <template
+      #actions="{ submit }"
+      v-if="formMode !== OfferingFormModes.Readonly"
+    >
       <footer-buttons
+        justify="space-between"
         :processing="processing"
-        primaryLabel="Submit"
-        @primaryClick="submit"
         @secondaryClick="cancel"
-      />
+        class="mx-0"
+      >
+        <template #primary-buttons="{ disabled }">
+          <span>
```

Comment:
IMO, its can be a div

Reply:
`span` is inline and `div` block-level, these buttons go side by side (inline), what would be the justification for it being a `div`?

Reply:
okay, but it's an entire block in the footer. right?

mm...IMO, these are examples of span, 
![image](https://user-images.githubusercontent.com/77353155/200914169-efac26eb-0b75-4e60-b0ef-28aad0eb6b52.png)
maybe I am missing anything


if it's inline? do we need a span here?

Reply:
"_okay, but it's an entire block in the footer. right?_", nope, not right, IMO. Please let me know if we need a call.
![image](https://user-images.githubusercontent.com/61259237/200918354-00b588a9-cc49-4aa7-8493-b6a4f4f28df5.png)

Reply:
no. I got it now, it's the extension of `#primary-buttons`. got it. then Inline makes sense. Thanks